examples/send_message.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+from avkapi import VK
+from avkapi.types.keyboard import Colours, ListOfButtons
+
+VK_CONFIRMATION_CODE = ""
+VK_CALLBACK_SECRET_KEY = None
+VK_ACCESS_TOKEN = "6dd495521793844f17510f6359445288fc6cc3dae00de6e1c6e8c173e7854322b3f989d58b20b8e1a3af8"
+
+loop = asyncio.get_event_loop()
+vk = VK(confirmation_code=VK_CONFIRMATION_CODE,
+        secret_key=VK_CALLBACK_SECRET_KEY,
+        access_token=VK_ACCESS_TOKEN,
+        loop=loop)
+
+keyboard = ListOfButtons(buttons=["Кнопка Белая", "Кнопка Синяя", "Кнопка Красная", "Кнопка Зеленая"],
```

Comment:
Non pythonic style has complicated logics. Please, use Button(**params) to create button.